public/js/app/components/MemoTable.jsx

```diff
@@ -1,51 +1,49 @@
 import React from 'react'
+import { seasonStr } from '../util/helpers'
+import Table from './Table'
 
 const row = (translation, courseCode, language, courseMemo) => {
```

Comment:
This is old code, but reacting to it nevertheless :) According to the clean code [chapter on naming](https://learning.oreilly.com/library/view/clean-code-a/9780136083238/chapter02.xhtml#ch2lev1sec8), functions should contain a verb, and I can get behind that. 
What do you think about that? Would `createRow` work for example?